official/recommendation/ncf_test.py

```diff
@@ -215,7 +216,24 @@ def test_end_to_end_keras_mlperf(self):
          '-distribution_strategy', 'off',
          '-log_steps', '100'])
 
+  @mock.patch.object(rconst, "SYNTHETIC_BATCHES_PER_EPOCH", 100)
+  def test_end_to_end_keras_1_gpu(self):
```

Comment:
you should probably skip the test if there are no GPUs available. otherwise it will just run with CPU i believe